.github/workflows/update-contribs.yml

```diff
@@ -2,19 +2,27 @@ name: Update Contributors
 
 on:
   workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - fix-workflow
   schedule:
     # Runs on the first of each month (see https://crontab.guru)
     - cron: "* * 1 * *"
 jobs:
+  # Reusable workflows can't be nested -
+  # https://stackoverflow.com/questions/71311762/reusable-workflows-should-be-referenced-at-the-top-level-jobs-uses-key-not
+  call-action:
```

Comment:
Docs suggests these should not be nested but i'm confused as to how to call it then. Right now the error is a YAML format error but previously it was the nesting causing issues.